Hardware/Computer.cs

```diff
@@ -180,11 +189,11 @@ public class Computer : IComputer {
       set {
         if (open && value != fanControllerEnabled) {
           if (value) {
-            Add(new TBalancer.TBalancerGroup(settings));
+            //Add(new TBalancer.TBalancerGroup(settings));
             Add(new Heatmaster.HeatmasterGroup(settings));
             Add(new Aquacomputer.AquacomputerGroup(settings));
           } else {
-            RemoveType<TBalancer.TBalancerGroup>();
+            //RemoveType<TBalancer.TBalancerGroup>();

```

Comment:
Should be undone.

Reply:
Sorry about this -> Thats clearly wrong -> Changed